syntax = "proto3";

service Booking {
  rpc GetBookingByUserId(UserId) returns (BookingData) {}
  rpc GetBookings(BookingEmpty) returns (stream BookingData) {}
  rpc CreateBooking(UserId) returns (BookingData) {}
}

message UserId {
  string id = 1;
}

message BookingData {
  string id = 1;
  string date = 2;
  repeated string movies = 3;
}

message BookingEmpty {}